crates/rspack_plugin_javascript/src/visitors/dependency/common_js_import_dependency_scanner.rs

```diff
@@ -232,6 +239,62 @@ impl<'a> CommonJsImportDependencyScanner<'a> {
   pub fn walk_expression(&mut self, expr: &Expr) {
     expr.visit_children_with(self);
   }
+
+  pub fn check_ident(&mut self, ident: &Ident) {
+    if is_reserved_word_in_strict(ident.sym.as_str()) {
+      if self.is_strict {
+        self.errors.push(
+          diagnostic!(
```

Comment:
Could you please change the diagnostic into a traceable error? This looks more beautiful to user end ;-). You can take this as a reference. https://github.com/web-infra-dev/rspack/blob/bfdc4279f1ae1f1f6332e74db963352d27c012a2/crates/rspack_plugin_javascript/src/utils/mod.rs#L63

Reply:
Also I'm planning to move other diagnostic to traceable errors as much as possible.

Reply:
Resolved, but only these new errors were modified to traceable, I will modify all the left errors of scanners in another pull request.